javascript/javascript-core/intro-js/introduction-js.md

```diff
@@ -0,0 +1,35 @@
+---
+author: abdullah
+tags:
+  - discussion
+
+type: normal
+category: discussion
+
+---
+
+# Introduction
+
+---
+
+## Content
+
+JavaScript is Everywhere!
+
+Every time you refresh your Twitter feed or search for a pair of skinny jeans on Amazon, you interact with JavaScript.
+
+For instance, the following code shows an alert that you might be familiar with 👖:
+
+```js
+alert("Warning! Jeans no longer fit.");
```

Comment:
the screenshot below says "Warning! 👖 no longer fit."` but the `alert` says `"Jeans"` (instead of the emoji)

Reply:
I thought i fixed that. Anyways, i have now.